lib/module.js

```diff
@@ -4,8 +4,6 @@ const consola = require('consola')
 const chalk = require('chalk')
 const { getTags, detectTags } = require('./tags')
 
-const logger = consola.withScope('@nuxtjs/amp')
```

Comment:
It's important to keed logs in scope for better readability. please undo changes about logger

Reply:
Yes, actually i let this PR go downhill, as i was making tests on my forked version and i removed things. Going to remove this and merge your refactor